test/regress/regress0/datatypes/issue5280-no-nrec.smt2

```diff
@@ -0,0 +1,7 @@
+; EXPECT: (error "")
```

Comment:
A suggestion: might be a good idea to duplicate this test with `dtNestedRec` turned on, and then to expect no errors. Up to you if that's necessary.

Reply:
This leads to a memout.  This actually crashed my machine.

This should not be a test, since --dt-nested-rec is not recommended.